parsers/UY.py

```diff
@@ -99,12 +106,12 @@ def fetch_data(
         )
 
 
-def fix_solar_production(dt: datetime, row: pd.Series) -> int:
+def fix_solar_production(dt: datetime, value: pd.Series) -> int:
     """sets solar production to 0 during the night as there is only solar PV in UY"""
-    if (dt.hour <= 5 or dt.hour >= 20) and row.get("value") != 0:
+    if (dt.hour <= 5 or dt.hour >= 20) and value != 0:
         return 0
     else:
-        return round(row.get("value"), 3)
+        return round(value, 3)
```

Comment:
This has a type error as pd.Series can't be passed to round.

Reply:
Good catch